README.md

```diff
@@ -68,18 +67,29 @@ Note: This is an example of a non-normative note.
 
 # Overview
 
-Serving GraphQL over HTTP provides the ability to use the full advantages of GraphQL with the rich feature set of HTTP. Carrying GraphQL in HTTP does not mean that GraphQL overrides existing
-semantics of HTTP but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
+Serving GraphQL over HTTP provides the ability to use the full advantages of GraphQL with the rich feature set of HTTP.
+Carrying GraphQL in HTTP does not mean that GraphQL overrides existing semantics of HTTP,
+but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
 
-GraphQL naturally follows the HTTP request/response message model providing a GraphQL request in an HTTP request and
-GraphQL response in an HTTP response.
+GraphQL naturally follows the HTTP request/response message model,
+providing a GraphQL request in an HTTP request and GraphQL response in an HTTP response.
 
 # URL
 
-A GraphQL server operates on a single URL and all GraphQL requests for a given service should be directed
-at this URL. Other protocols may also use that URL.
+A GraphQL over HTTP compliant server MUST designate at least one URL that handles GraphQL requests.
+
+It is allowed for the same GraphQL schema to be available on multiple URLs.
+
+All GraphQL operations that are available in the schema MUST be available on each designated URL.
+That means the client can send all their GraphQL requests to a single endpoint, regardless
```

Comment:
I think this is the reasoning for serving the entire schema on a single URL, it is valuable to mention that in the spec.

Reply:
I'd rather first stress that a dedicated URL has to support everything in "its" schema, so clients can send all requests based on that schema to the same URL.

And then point out that there may be multiple URLs, each serving a potentially (but not necessarily) different schema.

Reply:
> I'd rather first stress that a dedicated URL has to support everything in "its" schema, so clients can send all requests based on that schema to the same URL.

I tweaked the wording to stress that bit.

> And then point out that there may be multiple URLs, each serving a potentially (but not necessarily) different schema.

This was also suggested by @ghmcadams, i added a note saying that multiple URLs may exist.